src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+import { Post } from '../../types/post';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+interface PostInfoProps {
+  post: Post;
+}
+
+export const PostInfo: React.FC<PostInfoProps> = ({ post }) => {
+  const {
+    title, user, body, comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {(comments.length > 0)
```

Comment:
are those necessary?
```suggestion
      {comments.length > 0
```